src/FormTextarea.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Form from 'react-bootstrap/Form';
+
+const FormTextarea = (props) => {
+  const {controlId, label, rows, placeholder, subText, isRequired, value, handleChange} = props;
+  return (
+    <React.Fragment>
+      <Form.Group controlId={controlId}>
+        <Form.Label>{label}</Form.Label>
+        <Form.Control 
+          as="textarea"
+          rows={rows}
```

Comment:
Might be good to have a default value here in case we forget to pass it in?

FYI, this would be a good use case for Flow.js to ensure we pass in the props we need.